parser/cst.go

```diff
@@ -463,7 +463,10 @@ func NewCallStmt(name string, args []*Expr, with *WithClause, binds *BindClause)
 	}
 }
 
-func (cs *CallStmt) Breakpoint() bool {
+func (cs *CallStmt) Breakpoint(returnType Kind) bool {
+	if returnType == Kind(fmt.Sprintf("%s::%s", Option, "run")) {
```

Comment:
This is no longer necessary right?

Reply:
It is necessary now that I added the run option. Without it, `breakpoint` inside `with option {` gets treated as a breakpoint, and the debugger will break on that option (rather than just the `run`).

Reply:
I understand now, I was confusing this with trying to use option::run with the breakpoint instruciton.

Reply:
Also please let me know if you can think of a cleaner way to do this!